pages/dashboard.tsx

```diff
@@ -13,7 +13,7 @@ const Dashboard = () => {
         <ThresholdGraph />
       </div>
       <div className="flex flex-row w-full justify-start">
-        <NumberDisplay title="Today's Expected Output" value="10.23 kWh" />
+        <ExpectedOutput />
```

Comment:
Can we rename this component to something like `ExpectedTotalOutput`? just need some keyword like "total" in there to signify that it's an aggregate.